spec/models/room_message_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.describe RoomMessage, type: :model do
+  let(:room_message) { FactoryBot.build :room_message }
+
+  it 'is creatable using a factory' do
+    expect(room_message).to be_valid
+  end
+
+  it 'is invalid without a message' do
+    room_message.message = ''
+    expect(room_message).not_to be_valid
+  end
+
+  it 'json should contain the display name key' do
```

Comment:
Somehow unnecessary because you try to access this key in the next test, so that would fail in case the key does not exist. But fine to leave.